contrib/mod_tls_shmcache.c

```diff
@@ -564,7 +564,7 @@ static struct sesscache_data *sess_cache_get_shm(pr_fh_t *fh,
     "using shm ID %d for sesscache path '%s' (%u sessions)", sesscache_shmid,
     fh->fh_path, shm_sess_max);
 
-  data->sd_entries = (struct sesscache_entry *) (data + sizeof(struct sesscache_data));
+  data->sd_entries = (struct sesscache_entry *) (data + 1);
```

Comment:
Since `data` is defined to be a `struct sesscache_data *`, we don't need to use `sizeof`; we only need to increment by one, and let C naturally Do The Right Thing(tm).